app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express")
+const app = express()
+const { getTopics } = require("./controllers/topics")
+const { getArticleById } = require('./controllers/article_id')
+
+app.get('/api/topics', getTopics)
+
+app.get('/api/articles/:article_id', getArticleById)
+
+app.get('/*', (req, res) => {
+    res.status(404).send({ message: 'Not Found' });
```

Comment:
For line 10 - this will only currently work for `get` requests, however you could modify it to be `app.all(...` and this will cover all methods with an invalid endpoint